src/app/main-layout/main-layout.component.spec.ts

```diff
@@ -16,6 +18,9 @@ describe('MainLayoutComponent', () => {
         RouterTestingModule.withRoutes([]),
         TranslateModule.forRoot(),
       ],
+      providers: [
+        { provide: AuthService, useClass: MockAuthService }
+      ],      
```

Comment:
`ng lint` complains about the trailing spaces on that line.

![image](https://user-images.githubusercontent.com/38329766/142728566-d154ec51-915b-475d-8f54-601c782335b8.png)

Reply:
done.